pillar/components/release.sls

```diff
@@ -18,6 +18,7 @@
 release:
     product: LR2
     setup: cortx
+    install_type: base
```

Comment:
Please document what base means and other possible types.

Reply:
Yes, I will document this and other API details today during the day. Maybe in separate PR

Reply:
It would be better to add a small comment here itself

Reply:
It is no more needed due to this parameter was removed in the new version of this PR.